Scientific publications where Cell-ACDC was used
================================================

In the following publications authors used Cell-ACDC to analyse microscopy data:

1.	Al-Refaie, N. et al. *Fasting shapes chromatin architecture through an mTOR/RNA Pol I axis*. 
    **Nat. Cell Biol.** 1–15 (2024) DOI: `10.1038/s41556-024-01512-w <https://doi.org/10.1038/s41556-024-01512-w>`__.

2.	Lanz, M. C. et al. *Genome dilution by cell growth drives starvation-like proteome remodeling in mammalian and yeast cells*. 
    **Nat. Struct. Mol. Biol.** (2024) DOI: `10.1038/s41594-024-01353-z <https://doi.org/10.1038/s41594-024-01353-z>`__.

3.	Jordan Xiao, Jonathan J. Turner, Mardo Kõivomägi, & Jan M. Skotheim. *Whi5 hypo- and hyper-phosphorylation dynamics control cell cycle entry and progression*. 
    **bioRxiv** (2023) DOI: `10.1101/2023.11.02.565392 <https://doi.org/10.1101/2023.11.02.565392>`__.

4.	Vitacolonna, M. et al. *A multiparametric analysis including single-cell and subcellular feature assessment reveals differential behavior of spheroid cultures 
    on distinct ultra-low attachment plate types*. 
    **Front. Bioeng. Biotechnol.** 12, (2024) DOI: `10.3389/fbioe.2024.1422235 <https://doi.org/10.3389/fbioe.2024.1422235>`__.

5.	Vitacolonna, M. et al. *A Multiparametric Analysis Reveals Differential Behavior of Spheroid Cultures on Distinct Ultra-Low Attachment Plates Types*. 
    **bioRxiv** (2024) DOI: `10.1101/2024.03.26.586778 <https://doi.org/10.1101/2024.03.26.586778>`__.

6.	Roussou, R. et al. *Real-time assessment of mitochondrial DNA heteroplasmy dynamics at the single-cell level*. 
    **EMBO J.** 43, 5340–5359 (2024) DOI: `10.1038/s44318-024-00183-5 <https://doi.org/10.1038/s44318-024-00183-5>`__.

7.	Padovani, F. et al. *SpotMAX: a generalist framework for multi-dimensional automatic spot detection and quantification*. 
    **bioRxiv** (2024) DOI: `10.1101/2024.10.22.619610 <https://doi.org/10.1101/2024.10.22.619610>`__.

8.	Kukhtevich, I. et al. *The origin of septin ring size control in budding yeast*. 
    **bioRxiv** (2024) DOI: `10.1101/2024.07.30.605628 <https://doi.org/10.1101/2024.07.30.605628>`__.

9.	Chatzitheodoridou, D., Bureik, D., Padovani, F., Nadimpalli, K. V. & Schmoller, K. M. *Decoupled transcript and protein concentrations ensure histone homeostasis in different nutrients*. 
    **EMBO J.** 43, 5141–5168 (2024) DOI: `10.1038/s44318-024-00227-w <https://doi.org/10.1038/s44318-024-00227-w>`__.

10.	Chadha, Y., Kukhtevich, I. V., Padovani, F., Schneider, R. & Schmoller, K. M. *Single-cell imaging reveals a key role of Bck2 in budding yeast cell size adaptation to nutrient challenges*. 
    **bioRxiv** (2024) DOI: `10.1101/2024.10.04.616606 <https://doi.org/10.1101/2024.10.04.616606>`__.

11.	Seel, A. et al. *Regulation with cell size ensures mitochondrial DNA homeostasis during cell growth*. 
    **Nat. Struct. Mol. Biol.** 30, 1549–1560 (2023) DOI: `10.1038/s41594-023-01091-8 <https://doi.org/10.1038/s41594-023-01091-8>`__.

12.	Piñeiro López, C., Rodrigues Neves, A. R., Čavka, I., Gros, O. J. & Köhler, S. 
    *Segmentation of C. elegans germline nuclei*. **MicroPubl Biol.** (2023) 
    DOI: `10.17912/MICROPUB.BIOLOGY.001062 <https://doi.org/10.17912/MICROPUB.BIOLOGY.001062>`__.

13.	Chatzitheodoridou, D., Bureik, D., Padovani, F., Nadimpalli, K. V. & Schmoller, K. M. *Decoupling of transcript and protein concentrations ensures budding yeast histone homeostasis in different nutrient conditions*. 
    **bioRxiv** (2023) DOI: `10.1101/2023.01.26.525696 <https://doi.org/10.1101/2023.01.26.525696>`__.

14.	Freitag, M. et al. *Single-molecule experiments reveal the elbow as an essential folding guide in SMC coiled-coil arms*. 
    **Biophys. J.** 121, 4702–4713 (2022) DOI: `10.1016/j.bpj.2022.10.017 <https://doi.org/10.1016/j.bpj.2022.10.017>`__.

15.	Kukhtevich, I. V. et al. *Quantitative RNA imaging in single live cells reveals age-dependent asymmetric inheritance*. 
    **Cell Rep.** 41, (2022) DOI: `10.1016/j.celrep.2022.111656 <https://doi.org/10.1016/j.celrep.2022.111656>`__.